safesocket/safesocket_ps.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build linux || windows || darwin || freebsd
+// +build linux windows darwin freebsd
```

Comment:
openbsd?

Reply:
The OpenBSD build failure is why I pulled this out to its own file in PS2. In the first version I had it in a portable file and discovered that the ps package doesn't support OpenBSD. So oh well.